pycqed/instrument_drivers/meta_instrument/qubit_objects/QuDev_transmon.py

```diff
@@ -2519,20 +2491,19 @@ def find_RB_gate_fidelity(self, cliffords, nr_seeds, label=None,
 
         #Analysis
         if analyze:
-            pulse_length = self.gauss_sigma() * self.nr_sigma()
             if interleaved_gate is None:
-                rbma.RandomizedBenchmarking_Analysis(label=label,
-                                 qb_name=self.name,
-                                 T1=T1, T2=T2, pulse_length=pulse_length,
-                                 gate_decomp=gate_decomposition)
+                if classified_ro:
+                    ba3.BaseDataAnalysis()
```

Comment:
Why do we run the base analysis here? Don't we have an analysis for normal RB?

Reply:
This has been fixed in Proj_XLD. The code here is a preliminary version. 
BaseDataAnalysis here is now called PipelineDataAnalysis; it is the one that handles the pipe in analysis_v3, so this is how one runs any analysis in analysis_v3. The only part specific to each analysis is the pipeline.

Reply:
I see, and it gets the pipeline from the metadata of the hdf5.